airbyte-webapp/src/components/ReleaseStageBadge/ReleaseStageBadge.tsx

```diff
@@ -41,7 +41,7 @@ export const ReleaseStageBadge: React.FC<ReleaseStageBadgeProps> = ({ stage, sma
     );
 
   return tooltip ? (
-    <ToolTip control={badge} cursor="help">
```

Comment:
Reviewed this with @Upmitt and decided that it didn't make sense to show the help cursor here.